checks/shell_download_validate.go

```diff
@@ -401,7 +400,63 @@ func isPipUnpinnedDownload(cmd []string) bool {
 		}
 	}
 
-	return isInstalled
+	return isInstall
+}
+
+func isPythonCommand(cmd []string) bool {
+	for _, pi := range pythonInterpreters {
+		if isBinaryName(pi, cmd[0]) {
+			return true
+		}
+	}
+	return false
+}
+
+func isPythonPipInstall(cmd []string) bool {
```

Comment:
There are some false positives if someone genuinely tries to do `pip install *.whl` for `*.whl` for tests. Example: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/ci_build/builds/docker_cpu_pip.sh#L23

Reply:
I'm not sure if it's common at all for people to install a *.whl unless its in a testing context. Perhaps we should just exclude this.

Reply:
gotcha. Will remove this and maybe re-introduce it with more accuracy when working on https://github.com/ossf/scorecard/issues/613

Reply:
done.